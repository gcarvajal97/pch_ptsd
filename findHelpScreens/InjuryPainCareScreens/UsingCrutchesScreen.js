import React, { Component } from 'react';
import { View, Text, StyleSheet } from 'react-native';
import translate from '../../components/translateService';
import colors from '../../colors';

export default class UsingCrutchesScreen extends Component {
  render() {
    return (
      <View>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch4_p1')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch4_p2')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch4_p3')}</Text>

          <Text style={styles.paragraphTitle}>{translate('injuryPainCare.CH4_TITLE_1')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch4_t1_b1')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch4_t1_b2')}</Text>

          <Text style={styles.paragraphTitle}>{translate('injuryPainCare.CH4_TITLE_2')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch4_t2_b1')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch4_t2_b2')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch4_t2_b3')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch4_t2_b4')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch4_t2_b5')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch4_t2_b6')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch4_t2_b7')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch4_t2_b8')}</Text>

          <Text style={styles.paragraphTitle}>{translate('injuryPainCare.CH4_TITLE_3')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch4_t3_b1')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch4_t3_b2')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch4_t3_b3')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch4_t3_b4')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch4_t3_b5')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch4_t3_b6')}</Text>
                  
          <Text style={styles.paragraphTitle}>{translate('injuryPainCare.CH4_TITLE_4')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch4_t4_b1')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch4_t4_b2')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch4_t4_b3')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch4_t4_b4')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch4_t4_b5')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch4_t4_b6')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch4_t4_b7')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch4_t4_b8')}</Text>
                  
          <Text style={styles.paragraphTitle}>{translate('injuryPainCare.CH4_TITLE_5')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch4_t5_b1')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch4_t5_b2')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch4_t5_b3')}</Text>
      </View>
    );
  }
}

const styles = StyleSheet.create({
  paragraphTitle: {
    fontFamily: "avenir-heavy",
    fontSize: 20,
    alignSelf: 'stretch',
    textAlign: 'left',
    marginTop: 6,
    marginBottom: 3,
    color: colors.accent
  },
  paragraph: {
    textAlign: 'left',
    fontSize: 14,
    fontFamily: "avenir-medium",
    marginVertical: 2,
  },
  paragraphBold: {
    textAlign: 'left',
    fontSize: 14,
    fontFamily: "avenir-heavy",
    marginVertical: 2,
    marginLeft: 10
  }
});