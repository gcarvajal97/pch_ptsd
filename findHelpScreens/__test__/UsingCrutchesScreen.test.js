import React from 'react';
import UsingCrutchesScreen from '../InjuryPainCareScreens/UsingCrutchesScreen';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
    const tree = renderer.create(<UsingCrutchesScreen/>).toJSON();
    expect(tree).toMatchSnapshot();
}) 