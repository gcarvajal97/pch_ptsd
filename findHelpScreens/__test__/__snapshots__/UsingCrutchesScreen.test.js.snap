// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    For hip, leg, knee, ankle, or foot injuries, your healthcare provider may order crutches to decrease pain, speed recovery, and help with walking. Make sure that a trained healthcare professional fits your child for child-sized crutches and teaches your child how to use them.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Ask your healthcare provider for specific signs and symptoms that should alert you to call them right away. They may include the following:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Follow instructions provided by your healthcare provider (doctor or nurse practitioner) It is important that your child follow all of the special instructions given by your healthcare provider to make sure that your child heals completely. If you have any questions or concerns about the fit of your child's crutches or what should or should not do while on crutches, please contact your healthcare provider.
  </Text>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "color": "#2089DC",
        "fontSize": 20,
        "fontWeight": "bold",
        "marginBottom": 3,
        "marginTop": 6,
        "textAlign": "left",
      }
    }
  >
    General Tips:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Remind your child that crutches are not toys or weapons.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Even if your child may be feeling better, make sure to use the crutches as long as your healthcare provider told you to use them. If you start putting weight on the leg too soon, you might cause a new injury or increase the recovery time.
  </Text>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "color": "#2089DC",
        "fontSize": 20,
        "fontWeight": "bold",
        "marginBottom": 3,
        "marginTop": 6,
        "textAlign": "left",
      }
    }
  >
    Precautions - Your Child Should:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Be very careful on slick or wet surfaces (like in the kitchen and the bathroom).
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Be careful of throw rugs that can slip. They should be removed.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Never hop around holding on to furniture; it may slide or fall.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Keep the crutches nearby so they are always in reach.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Wear low-heeled shoes (like sneakers) that will not slip off.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Be careful of ramps or slopes. It is harder to walk with crutches on ramps or slopes.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Do not remove any parts of the crutches, including the rubber tips.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - If they feel uncomfortable on crutches, ask someone to help. For example, someone can carry their books.
  </Text>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "color": "#2089DC",
        "fontSize": 20,
        "fontWeight": "bold",
        "marginBottom": 3,
        "marginTop": 6,
        "textAlign": "left",
      }
    }
  >
    Helpful Hints
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - A bedside toilet may be used.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Ask teachers in school to let your child out of class a little early to avoid crowds on the stairs.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Keep the 'bad' leg up on a stool while sitting.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Carry schoolbooks in a backpack to leave both hands free and ask a friend to carry it for you, if possible.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Avoid leaning on the underarm pieces.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - If your child's school is more than two stories tall, ask if your child can have a key to the elevator.
  </Text>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "color": "#2089DC",
        "fontSize": 20,
        "fontWeight": "bold",
        "marginBottom": 3,
        "marginTop": 6,
        "textAlign": "left",
      }
    }
  >
    Make sure to ask your healthcare provider:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - How long will my child need to be on crutches?
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Can my child bear any weight on his injured leg?
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - What are the limits to my child's activity while he is on crutches?
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - What are the limits on my child's activity once he is off of his crutches?
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - When can my child return to school?
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - When can my child return to gym class? to contact sports? to biking? or to rough play?
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Will my child need physical therapy?
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Will this injury affect my child's growth?
  </Text>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "color": "#2089DC",
        "fontSize": 20,
        "fontWeight": "bold",
        "marginBottom": 3,
        "marginTop": 6,
        "textAlign": "left",
      }
    }
  >
    When to call your healthcare provider:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Your child has increasing pain and pain medicine recommended or prescribed by your healthcare provider is not helping.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Your child feels numbness or tingling in his arms.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - If you have any other concerns.
  </Text>
</View>
`;
